File: ex9vars.py
# ex9: Variables, Python

# Single value, "immutable"
foo = 1
bar = foo  # Copy value
bar = 99

print(foo, bar)  # 1 99

# Values in container (list),  "mutable"
foo = [1, 2, 3]
bar = foo     # Point b at SAME list
bar[0] = 99

print(foo, bar)   # [99, 2, 3] [99, 2, 3] !!!

File: ex9vars.m
% ex9: Variables, MATLAB

% Single value
foo = 1;
bar = a;  %#ok Copy value
bar = 99;

disp([foo bar]);  % 1 99

 % Values in container (array)
foo = [1, 2 ,3];
bar = foo;     % Copy array
bar(1) = 99;

disp([foo bar]);   % [1 2 3]  [99 2 3]
